setup/src/python/app/scripts/language-specific-setup.sh

```diff
@@ -52,22 +37,3 @@ REQUIREMENTS="./app/tests/requirements.txt"
 if [ -f $REQUIREMENTS ]; then
     pip3 install -r $REQUIREMENTS
```

Comment:
I like the refactoring - but how much do we want to do with Python+offline?

I see two approaches:

- Leave it as it is now - then the "pip3 install" commands will likely fail, right?
- Add a check `if [[ -z "${VELOCITAS_OFFLINE}" ]]; then` around all `pip3 install` commands, assuming that if you use this in offline mode there should be no need for the pip commands, everything should be already installed, or? (Not saying that the assumption is true)

Reply:
Good point, added a check for offline around the installations